### DISCUSSION TITLE
Welcome to AuctionPro Discussions!

### DISCUSSION THREAD

**Discussion by Jaynil1611** [Jaynil1611/AuctionPro#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kooly** [comment]
Hi Jaynil,

Thanks for sharing that code.

I wonder where I can find http://localhost:3000/api/Users  API or I have to create one myself? and why you have api and connecting aws database in the same time

**Reply by Jaynil1611** [reply]
Actually, we had used amazon was for this project. But, now the credits have been exhausted so the database is not working.

**Reply by kooly** [reply]
and what about the queries you  made in database.js is it connecting the same aws database?

**Reply by Jaynil1611** [reply]
Yes

**Reply by kooly** [reply]
Thanks
